checks/evaluation/security_policy.go

```diff
@@ -39,11 +40,14 @@ func SecurityPolicy(name string, dl checker.DetailLogger, r *checker.SecurityPol
 		}
 		if msg.Type == checker.FileTypeURL {
 			msg.Text = "security policy detected in org repo"
+			dl.Info(&msg)
 		} else {
-			msg.Text = "security policy detected"
+			// Security policy detected in current repo, return immediately since it has higher priority.
+			msg.Text = "security policy file detected in current repo"
+			dl.Info(&msg)
+			return checker.CreateMaxScoreResult(name, "security policy file detected in current repo")
 		}
-		dl.Info(&msg)
 	}
-
-	return checker.CreateMaxScoreResult(name, "security policy file detected")
```

Comment:
can keep the message the same in all cases. Users who want to know the details can inspect the `details` field.

Reply:
Addressed, thanks